src/components/Colors/Colors.stories.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from "react";
+
+import { boolean, text } from "@storybook/addon-knobs";
+import cssVariables from "../../helpers/CSSVariablesHelper";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import UIDocCard from "./UIDocCard";
+import Card from "../Card/Card";
+import List from "../List/List";
+import Heading from "../Heading/Heading";
+import { ListTypes } from "../List/ListTypes";
+
+export default {
+    title: "Colors",
+    decorators: [(storyFn) => <div style={{ padding: "2%" }}>{storyFn()}</div>],
```

Comment:
I forgot how but you can make this a global decorator for all stories so you don't have to duplicate this in every story.

Reply:
Try adding this to `.storybook-react/config.js`.

```
export const CenterDecorator = storyFn => <div style={styles}>{storyFn()}</div>;
addDecorator(CenterDecorator);
```

Reply:
🙇‍♂️ thank you!

Do you know I _unset_ it for certain stories? (`Hero` and `Breadcrumbs` should span full width, for example.)